app/models/identity_tijuana/user.rb

```diff
@@ -51,8 +51,8 @@ def import(sync_id)
           subscriptions: []
         }
 
-        member_hash[:custom_fields].push({name: 'deceased', value: 'true'}) if (has_tag('deceased'))
-        member_hash[:custom_fields].push({name: 'rts', value: 'true'}) if (has_tag('rts'))
+        member_hash[:custom_fields].push({name: 'deceased', value: 'true'}) if has_tag('deceased')
+        member_hash[:custom_fields].push({name: 'rts', value: 'true'}) if has_tag('rts')
```

Comment:
Totally not actually worth addressing here, but I (being highly anally retentive) would have done a `git rebase -i` and made these two changes a fixup for the commit a few back that intro'ed these changes. :)

Reply:
Nice to finally meet someone more anally retentive than myself.